libsplinter/src/rest_api/auth/identity/mod.rs

```diff
@@ -33,10 +33,8 @@ use super::AuthorizationHeader;
 pub enum Identity {
     /// A custom identity
     Custom(String),
-    #[cfg(feature = "cylinder-jwt")]
     /// A Cylinder public key
     Key(PublicKey),
```

Comment:
This will need to be a String to prevent the cylinder dependency.

Reply:
Updated